src/providers/MeetingProvider/index.tsx

```diff
@@ -35,6 +36,10 @@ interface Props {
    * Check `meetingManager` prop documentation for more information.
    */
   meetingManager?: MeetingManager;
+  /**
+   * Whether or not to enable Amazon Chime SDK for JavaScript meeting events.
+   */
+  enableMeetingEvents?: boolean;
```

Comment:
- `meetingEventEnabled` to follow the naming convention, e.g., `simulcastEnabled` (Or just remove this boolean property if you follow Michael's suggestion below.)

Reply:
Yup I followed Michael's suggestion below. But thanks for the examples, have removed the flag now.